src/conversions.jl

```diff
@@ -556,6 +556,25 @@ function convert_arguments(
     m = normal_mesh(to_vertices(vertices), to_triangles(indices))
     (m,)
 end
+                        
+################################################################################
+#                                   <:Arrows                                   #
+################################################################################
+
+# Allow the user to pass a function to `arrows` which determines the direction
+# and magnitude of the arrows.  The function must accept `Point2f` as input.
+# and return Point2f or Vec2f or some array like structure as output.
+function convert_arguments(::Type{<: Arrows}, x::AbstractVector, y::AbstractVector, f::Function)
+    points = Point2f.(x, y')
```

Comment:
Shouldn't that be `Point2f.(x, y)`?

Reply:
No, arrows always creates a matrix - the eventual positions are `Point2f.(x, y')` anyway. Plus the inputs are AbstractVectors so we know that we have to make the grid.